docs/Docker-install-windows.md

```diff
@@ -0,0 +1,206 @@
+Docker Compose is used to create a full installation of fAIr (with redis, worker, postgis database, api and frontend), all in one Docker container. This setup is suitable for development. For production it is not recommended. ***TODO*** What is production? Is it the version that executes on the web site?
+
+## Installation of fAIr using Docker
+
+In the following, four directories are created: ```fair, ramp-code, ramp``` and ```training```, for the fAIr code, the model code, the model variables, and temporary data used during fine-tunining training of the (RAMP) model. It is a good idea to create these directories in the same, new, directory, created for this installation of fAIr and its companions.
+
+1. Check your graphics card
+
+    It is highly recommended to use a graphics card to run fAIr. It might not work with CPU only. (You can set up and test from bottom of this document). ***TODO*** What does this mean? NVIDIA graphics cards are tested. The bottom of this document is about a tile server, whatever that is. But it is not a GPU test?! ***TODO*** Does this mean that it works for NVIDIA cards but not for cards from other brands? 
```

Comment:
Tile server is different , this is solely meant for serving your local geotiff to fAIr Read more here : https://developmentseed.org/titiler/ 

On this doc , When we say NVIDIA cards are tested it means : Developers have tested fAIr with NVIDIA Graphics . fAIr basemodels might not work with other graphics or might work we haven't tested them yet. 

It doesn't mean it won't work , it can solely be setup in CPU only too but may not be ideal for trainings ! I am pretty sure to be able to setup basemodels like RAMP in other graphics card will be hell of pain

Reply:
At least normally, I put the TODO comments after the text that I think is problematic. Here my problem is with the text "(You can set up and test from bottom of this document)." To me it looks like it is saying "At the bottom of this text there is an instruction for how to set up and test your graphics card." But there is no such instruction at the bottom of this text. So I am clueless, and I think many other readers will also find this difficult to understand. 

(I understand the other part.)

Reply:
You are right , we can include following link for setup  https://www.nvidia.com/Download/index.aspx 
For testing nvidia-smi will be enough if it returns you have set up your drivers correctly if not either you don't have graphics card or drivers are not setup